extensions/amp-story/1.0/media-pool.js

```diff
@@ -807,7 +807,7 @@ export class MediaPool {
     return this.enqueueMediaElementTask_(poolMediaEl, new PauseTask()).then(
       () => {
         if (rewindToBeginning) {
-          // We add a 10 second delay to rewinding as sometimes this causes an
+          // We add a 100 second delay to rewinding as sometimes this causes an
           // interlacing/glitch/frame jump when a new video is starting to play.
           // A 0 delay isn't enough as we need to push the "seeking" event
           // to the next tick of the event loop.
```

Comment:
Can you create a ticket and reference it here, to fix the browser frame management during page transitions?

Reply:
done. added GitHub issue and TODO note on description https://github.com/ampproject/amphtml/issues/38595